src/YoTennis/Models/GameModel.cs

```diff
@@ -9,22 +9,66 @@ namespace YoTennis.Models
     public class GameModel
     {
         public List<GameEvent> Events { get; private set; }
-        public State CurrentState { get; private set; }
+        public State CurrentState { get; private set; } = new State();
 
-        public void AddPoint(Player GetPointPlayer, PointKind PointKind, ServeSpeed ServeSpeed)
+        public void AddPoint(Player getPointPlayer)
         {
-            //AddPoint
-            /*
-            if (GetPointPlayer == Player.First)
+            var score = CurrentState.ScoreInGame.Score = CurrentState.ScoreInGame.Score + PlayerScore.ForPlayer(getPointPlayer, 1);
+            if (score.FirstPlayer >= 4 && score.SecondPlayer + 1 < score.FirstPlayer)
+            {
+                PlayerWonGame(Player.First);
+            }
+            else if (score.SecondPlayer >= 4 && score.FirstPlayer + 1 < score.SecondPlayer)
             {
-                CurrentState.ScoreInGame.Score.FirstPlayer ++;
+                PlayerWonGame(Player.Second);
             }
             else
             {
-                CurrentState.ScoreInGame.Score.SecondPlayer++;
+                CurrentState.ServePositionOnTheCenter = CurrentState.ServePositionOnTheCenter == ServePositionOnTheCenter.Left ?
+                    ServePositionOnTheCenter.Right : ServePositionOnTheCenter.Left;
+                CurrentState.SecondServe = false;
             }
+        }
+
+        private void PlayerWonGame(Player player)
+        {
+            var currentSetIndex = CurrentState.ScoreInSets.Count - 1;
+
+            var currentSet = CurrentState.ScoreInSets[currentSetIndex];
+            var currentSetScore = currentSet.Score;
+
+            currentSetScore = currentSetScore + PlayerScore.ForPlayer(player, 1);
+            currentSet.Score = currentSetScore;
+
+            if (currentSetScore.FirstPlayer >= 6 && currentSetScore.SecondPlayer + 1 < currentSetScore.FirstPlayer)
```

Comment:
even if it's well-known constant prescribed in the game rules, I'd suggest to move '6' into a constant. Define a class like `public static class TennisConstants { ... public const int GamesInSet = 6; ... }`